e2e/proposer-code-1.3.sh

```diff
@@ -0,0 +1,8 @@
+#! /bin/bash
```

Comment:
This file is missing a copyright notice.
Also if possible, we should use `/bin/sh` and aim for the script to be POSIX compatible instead of requiring bash.
```suggestion

#!/bin/sh
```